tests/test_io.py

```diff
@@ -822,7 +822,7 @@ def test_sqlalchemy_selectable() -> None:
             class Base(metaclass=sqlalchemy.orm.decl_api.DeclarativeMeta):
                 __abstract__ = True
 
-            class Temp(Base):
+            class Temp(Base):  # type: ignore[misc]
```

Comment:
can you add a comment here as to why the `# type: ignore` is needed?

Reply:
I honestly don't understand why - I added the error reported by mypy